loadbalancer/etcd/client.go

```diff
@@ -0,0 +1,63 @@
+package etcd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreos/go-etcd/etcd"
+)
+
+// Client is a wrapper arround the etcd client
+type Client interface {
+	// GetEntries will query the given prefix in etcd and returns a set of entries.
+	GetEntries(prefix string) ([]string, error)
+	// WatchPrefix starts watching every change for given prefix in etcd. When an
+	// change is detected it will populate the responseChan when an *etcd.Response.
+	WatchPrefix(prefix string, responseChan chan *etcd.Response)
+}
+
+type client struct {
+	*etcd.Client
+}
+
+// NewClient returns an *etcd.Client with a connection to the named machines.
+// It will return an error if a connection to the cluster cannot be made.
+// The parameter machines needs to be a full URL with schemas.
+// e.g. "http://localhost:4001" will work, but "localhost:4001" will not.
+func NewClient(machines []string, cert, key, caCert string) (*client, error) {
```

Comment:
One last thing, this now returns an unexported type, which golint doesn't like. Can you please return a `(Client, error)` instead?